azure-pipelines.yml

```diff
@@ -51,6 +51,8 @@ steps:
 - script: python -m cimetrics.github_pr
   env:
     GITHUB_TOKEN: $(GITHUB_TOKEN)
+    AZURE_BLOB_URL: $(AZURE_BLOB_URL)
+    AZURE_WEB_URL: $(AZURE_WEB_URL)
```

Comment:
It'd be really nice to derive the second one from the first one, but I haven't worked out how to do it.